ui/src/dashboards/components/DashboardHeader.tsx

```diff
@@ -145,11 +146,14 @@ class DashboardHeader extends Component<Props> {
   }
 
   private get dashboardSwitcher(): JSX.Element {
-    const {names, activeDashboard} = this.props
+    const {dashboardLinks, activeDashboardLink} = this.props
 
-    if (names && names.length > 1) {
+    if (dashboardLinks.length > 1) {
```

Comment:
should we protecting against the possibility of dashboarLinks being undefined here?

Reply:
The `Props` interface should guarantee `dashboardLinks` is always an array.